test/parallel/test-child-process-detached.js

```diff
@@ -1,21 +1,21 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
-var path = require('path');
+const common = require('../common');
+const assert = require('assert');
+const path = require('path');
 
-var spawn = require('child_process').spawn;
-var childPath = path.join(common.fixturesDir,
+const spawn = require('child_process').spawn;
+const childPath = path.join(common.fixturesDir,
                           'parent-process-nonpersistent.js');
```

Comment:
nit: Can you indent this line 2 characters so that the arguments to `path.join` are still aligned?